includes/functions.inc.php

```diff
@@ -26,39 +26,30 @@ function input_convert($str) {
 }
 
 
-function format_ago($time, $ago = false) {
```

Comment:
If you want to remove the extra argument you also have to remove it from [overview.php#L88](https://github.com/erikdubbelboer/phpRedisAdmin/blob/961e1d5ae94152b7494048b2df5e5f1a6e01b574/overview.php#L88) and maybe add an `echo ' ago';` after the `format_ago` so it still looks nice.